sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -761,11 +776,47 @@ def data_api_service_descriptor(self):
   def state_api_service_descriptor(self):
     return None
 
+  def logging_api_service_descriptor(self):
+    return None
+
+
+class BasicLoggingService(beam_fn_api_pb2_grpc.BeamFnLoggingServicer):
```

Comment:
Can you list the new services added in the description of this PR?

Reply:
Done.